client/state/page/actions.js

```diff
@@ -0,0 +1,46 @@
+import {
+	PAGE_LINK_ADD,
+	PAGE_META_ADD,
+	PAGE_TITLE_SET,
+	PAGE_UNREAD_COUNT_SET
+} from 'state/action-types';
+
+export function setTitle( title ) {
+	return {
+		type: PAGE_TITLE_SET,
+		title
+	};
+}
+
+export function addLink( link ) {
+	return {
+		type: PAGE_LINK_ADD,
+		link
+	};
+}
+
+export function addMeta( meta ) {
+	return {
+		type: PAGE_META_ADD,
+		meta
+	};
+}
+
+export function setDescription( description ) {
+	let meta = {
```

Comment:
Typically, we prefer to use `let` only in cases where we know we'll reassign the value of a variable, and `const` in all other cases.